.pre-commit-config.yaml

```diff
@@ -33,6 +33,12 @@ repos:
   - id: check-merge-conflict
   - id: debug-statements
 
+- repo: https://github.com/ansys/pre-commit-hooks
+  rev: v0.2.4 # `strings.properties` needs to be manually updated
+  hooks:
+    - id: add-license-headers
+      args: ["--custom_copyright", "ANSYS, Inc. All rights reserved."]
```

Comment:
Why does it have to be custom?

Reply:
Thanks for having pointing this out. I removed this argument.